src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
You could have a better convention such as dateTime. "by" alone doesn't have any meaning.

Reply:
Hi! Thanks for the comment, here I did not implement "by" as a datetime object for deadline:) Thus, it is kept as a string. However, I may change the name of the variable.

Reply:
so sorry I thought this was my iP, please ignore my replies if any:)